x/concentrated-liquidity/README.md

```diff
@@ -766,56 +968,36 @@ Now, we update the swap state as follows:
 
 - `liquidity` is set kept the same as we did not cross any initialized tick.
 
-- `feeGrowthGlobal` is updated to 0.0025 + (2_500 \* 0.01 / 10_000) = 0.005 because we assumed 1% swap fee.
+- `feeGrowthGlobal` is updated to 0.0025 + (2_500 \* 0.01 / 10_000) = 0.005
+  because we assumed 1% swap fee.
 
-As a result, we complete the swap having swapped 5_000 tokens one in for 22_500 tokens zero out.
-The tick is now at 70 and the current liquidity at the active tick tracked by the pool is 11_000.
-The global fee growth per unit of liquidity has increased by 50 units of token one.
-See more details about the fee growth in the "Fees" section.
+As a result, we complete the swap having swapped 5_000 tokens one in for 22_500
+tokens zero out. The tick is now at 70 and the current liquidity at the active
+tick tracked by the pool is 11_000. The global fee growth per unit of liquidity
+has increased by 50 units of token one. See more details about the fee growth
+in the "Fees" section.
 
 TODO: Swapping, Appendix B: Compute Swap Step Internals and Math
 
 ## Range Orders
 
-> As a trader, I want to be able to execute ranger orders so that I have better control of the price at which I trade
+> As a trader, I want to be able to execute ranger orders so that I have better
+control of the price at which I trade
 
 TODO
 
 ## Fees
 
-> As a an LP, I want to earn fees on my capital so that I am incentivized to participate in the market making actively.
-
-For our balancer style pools, fees go back directly to the pool to benefit LPs.
-For a concentrated liquidity pool, this is no longer possible due to the non-fungible property
-of positions. As a result, there is a different accumulator-based mechanism for keeping track
-of and storing the fees.
-
-First, note that the fees are collected in tokens themselves rather than in units of liquidity.
-Thus, we need two accumulators for each token.
-
-TODO: explain the `accum` package and how it is used in CL.
-Reference these papers:
+> As a an LP, I want to earn fees on my capital so that I am incentivized to
+participate in active market making.
 
-- [Scalable Reward Distribution](https://uploads-ssl.webflow.com/5ad71ffeb79acc67c8bcdaba/5ad8d1193a40977462982470_scalable-reward-distribution-paper.pdf)
-- [F1 Fee Distribution](https://drops.dagstuhl.de/opus/volltexte/2020/11974/pdf/OASIcs-Tokenomics-2019-10.pdf)
+In Balancer-style pools, fees go directly back into the pool to benefit all LPs pro-rata.
+For concentrated liquidity pools, this approach is no longer feasible due to the
+non-fungible property of positions. As a result, we use a different accumulator-based
+mechanism for tracking and storing fees.
```

Comment:
I think linking the papers removed above and our `osmoutils/accum` package itself here would be helpful